Source/GRChomboCore/BoundaryConditions.cpp

```diff
@@ -141,6 +142,23 @@ void BoundaryConditions::params_t::read_params(GRParmParse &pp)
 
     if (reflective_boundaries_exist)
     {
+        int symmetry_correction = 0;
+        pp.load("symmetry_correction", symmetry_correction, 0);
```

Comment:
I would actually be tempted just to remove the `symmetry_correction` variable, and always work out this symmetry factor. I can't imagine going forward that someone will not want to use it. This also means we don't need to change all the params files for another variable that people will just ignore or find confusing.

Reply:
(Also just for reference from a code convention perspective it should be a bool called `uses_symmetry_correction` or something like that)